lab5.md

```diff
@@ -66,10 +66,10 @@ functions:
       write_debug: true
 ```
 
-* Build, push and deploy the function with
+* Deploy the function
 
 ```
-$ faas-cli up -f ./issue-bot.yml
```

Comment:
Why did you remove `faas-cli up`?

Reply:
@alexellis This is getting confusing but started from this discussion:
https://github.com/openfaas/workshop/pull/78#discussion_r224955735

I requested to revert the change:
https://github.com/openfaas/workshop/pull/78#discussion_r224955904

But actually, this instruction is creating the function right before this place so it actually needs the `faas-cli up`. If that's true, https://github.com/openfaas/workshop/pull/78/commits/d6e5d164962f8f625b3c9d933fdaa2118ea5a371 needs to be reverted. Alex, could you confirm?

@toshi0607 sorry about my misleading comment but could you revert the commit after Alex confirms?

Reply:
@toshi0607 Sorry about the totally late response. Could you please revert this to `$ faas-cli up -f ./issue-bot.yml` ? So that it matches the original version.
Sorry about the back and forth 😢